mkdocs.yml

```diff
@@ -55,7 +55,11 @@ nav:
     - Installation: GettingStarted/Installation.md
     - Configuration:
       - Pelorus Core: GettingStarted/configuration/PelorusCore.md
-      - Pelorus Exporters: GettingStarted/configuration/PelorusExporters.md
+      - Pelorus Exporters:
+        - Overview: GettingStarted/configuration/PelorusExporters.md
+        - Deploy Time: GettingStarted/configuration/ExporterDeploytime.md
+        - Commit Time: GettingStarted/configuration/ExporterCommittime.md
+        - Failure Time: GettingStarted/configuration/ExporterFailure.md
```

Comment:
it is **failure time** or just **failure**?

Reply:
In the docs we have elsewhere "Failure Time Exporter", e.g.:

"The job of the failure time exporter is to capture the timestamp at which a failure occurs in a production environment and when it is resolved."

"Failure Time Exporter may be deployed with one of three backends, such as JIRA, GitHub Issues"

So I think it should be failure time ?